### DISCUSSION TITLE
Make the Private Key Exportable

### DISCUSSION THREAD

**Discussion by jag-sandhu** [polymind-inc/acmebot#558] [Q&A] [answered]
Hi All,

is there any setting that can be added to the config to make the private key exportable?  

Thanks,

**Comment by shibayan** [comment, marked as answer]
There is no option to explicitly specify Exportable in the certificate issuance policy, but isn't Key Vault enabled to export private keys by default?

**Reply by jag-sandhu** [reply]
ah ok.
Key vault does export the private key.  I can just use local cert services to add a password.
Thanks
